packages/react-dom/src/__tests__/ReactDOMFizzServer-test.js

```diff
@@ -8,7 +8,12 @@
  */
 
 'use strict';
-import {replaceScriptsAndMove, mergeOptions} from '../test-utils/FizzTestUtils';
+import {
+  replaceScriptsAndMove,
+  mergeOptions,
+  stripExternalRuntimeInString,
+  stripExternalRuntimeInNodes,
```

Comment:
That makes sense - I have removed `stripExternalRuntimeInString` and changed the implementation to not emit the external runtime when rendering static documents.

I kept `stripExternalRuntimeInNodes` since it seemed to be more compatible with the spirit of tests, but happy to change this as well